www/src/pages/blog/astro-018.astro

```diff
@@ -10,17 +10,16 @@ import Shell from '../../components/Shell.astro';
 import { mediaQueries } from '../../config.js';
 
 let title = 'Astro 0.18 Release';
-let description = `Responsive partial hydration • Solid.js support • Lit SSR support • Named slots • Global style support • and more!`;
+let description = `Introducing: Responsive partial hydration • Solid.js support • Lit SSR support • Named slots • Global style support • and more!`;
```

Comment:
does the rss feed generator support describing updates to posts? 🤔

Reply:
how do you mean?

Reply:
As in, the rss feed (which i now realize is hand-generated if it exists) should update to the new post title and add a tag describing when the post was last updated

Reply:
Ah yes it should, but right now the `rss()` feed is only available inside of `getStaticPaths()` for (not very good) reasons.